### PR TITLE
[33482] System send mail to customer with order

### PR DESCRIPTION
**Pull request by ghost**
[33482](https://edu-redmine.sun-asterisk.vn/issues/33482)
image
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: app/mailers/order_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class OrderMailer < ApplicationMailer
+  default from: "Unistore Pro"
```
Comment (tham-1781) [inline app/mailers/order_mailer.rb:2]: thêm 1 line trống đi chú

File: app/views/order_mailer/send_order.html.erb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<h3>Unistore Pro</h3>
+<p>
+    <%= t("mails.dear") %> <b><%= @order.customer_name %></b>,
```
Comment (tham-1781) [inline app/views/order_mailer/send_order.html.erb]: format lại file này

Comment (ghost) [reply app/views/order_mailer/send_order.html.erb]: dạ anh zai

File: app/mailers/order_mailer.rb (hunk 1 above)
Comment (ghost) [reply app/mailers/order_mailer.rb:2]: ok anh zai

**Pull request closed without merging**